salt/modules/virt.py

```diff
@@ -2527,9 +2684,15 @@ def _set_loader(node, value):
     def _set_nvram(node, value):
         node.set("template", value)
 
-    def _set_with_mib_unit(node, value):
+    def _set_with_byte_unit(node, value):
         node.text = str(value)
-        node.set("unit", "MiB")
+        node.set("unit", "bytes")
+
+    def _get_with_unit(node):
+        unit = node.get("unit", "KiB")
+        unit = unit if unit != "bytes" else "b"
```

Comment:
That looks strange, why not the following?
```suggestion
        unit = node.get("unit", "b")
```